svg.c:240: warning: incompatible implicit declaration of built-in function ‘execlp’

--- svg.c
+++ svg.c
@@ -10,6 +10,7 @@
 #include <limits.h>
 #include <sys/stat.h>
 #include <sys/wait.h>
+#include <unistd.h> /* execlp */
 
 #ifndef _MSC_VER
 #include <X11/Intrinsic.h>
